phpstan-baseline.neon

```diff
@@ -1,2 +1,6 @@
 parameters:
-	ignoreErrors:
+  ignoreErrors:
+  -
+    message: "#^Call to function is_null\\(\\) with null will always evaluate to true\\.$#"
+    path: lib/Serializer/functions.php
+    count: 1
```

Comment:
This message is emitted because of line 204
```
    } elseif (!is_null($value)) {
        throw new \InvalidArgumentException('The writer cannot serialize values of type: '.gettype($value));
    }
```
The if-else cases already processed all the possible types according to the type declaration of `$value` except for `null`. So PHPstan thinks that `$value` must be `null` at that point, and so we can never get that `InvalidArgumentException`

But someone could pass some other unusual type of value to `standardSerializer` The type would not fit with the PHPdoc declared set of types, but the PHP run-time is not going to complain. So "catch-all" exceptions like that are actually valid/useful.